mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -745,7 +812,7 @@ func resourceStorageBucketCreate(d *schema.ResourceData, meta interface{}) error
 	// to make sure it exists before moving on
 	err = transport_tpg.Retry(transport_tpg.RetryOptions{
 		RetryFunc: func() (operr error) {
-			_, retryErr := config.NewStorageClient(userAgent).Buckets.Get(res.Name).Do()
+			_, retryErr := config.NewStorageClient(userAgent).Buckets.Get(res.Name).Projection("full").Do()
```

Comment:
What's the purpose of this change? (Note it would be the only change to the GA provider)

Reply:
Default projection does not contains ipFilter and full projection is required only for ipFilter so I added beta version guard.